content/fr/shift-left.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Shift Left
```

Comment:
Beaucoup d'interrogation autour du terme, la littérature et les articles francais ne le traduire pas, mais j'hesite entre : "Shift Left",  "approche Shift Left", "Methodologie Shift Left"

Reply:
Je propose "Décalage vers la gauche" ou "Décalage à gauche".

Reply:
Moi j'aime bien "Approche Shift Left"
"Décalage vers la gauche" n'est utilisé par personne, et si on cherche ces termes sur google on tombe sur des articles qui parlent de décalages de bits en programmation. "Approche Shift Left" donne des résultats pertinents.

Reply:
Nous avons traduit tous les autres termes en français jusque là.
Je propose de faire de même pour ce terme en conservant "Shift left" entre parenthèses.
De mon côté, j'ai déjà utilisé "décalage vers la gauche" avec quelques clients.
Ensuite, lors d'une recherche avec "décalage vers la gauche DevOps", je trouve des résultats dans ma bulle.

Reply:
OK, ça me va

Reply:
fix in https://github.com/cncf/glossary/pull/2956/commits/95a67215a8afdd86e0941c28204a28466bc805b6